package.json

```diff
@@ -22,8 +22,7 @@
     "core-js": "^3.8.0",
     "http-status-codes": "^1.4.0",
     "launchdarkly-js-client-sdk": "^2.18.3",
-    "moment": "^2.29.1",
-    "moment-timezone": "^0.5.33",
+    "lodash": "^4.17.20",
```

Comment:
Lodash was already imported by other libraries we use, so this doesn't add anything (but it is explicit that we need it).